temporian/core/operators/since_last.py

```diff
@@ -32,10 +32,14 @@ class SinceLast(Operator):
     def __init__(
         self,
         input: EventSetNode,
+        steps: int,
         sampling: Optional[EventSetNode] = None,
     ):
         super().__init__()
 
+        if steps <= 0:
+            raise ValueError("Invalid value received: {steps=}")

```

Comment:
would be nice to specify in the error that steps need to be >=1

Reply:
Totally 😄